ledger/acctupdates.go

```diff
@@ -842,7 +843,10 @@ func (aul *accountUpdatesLedgerEvaluator) LookupAsset(rnd basics.Round, addr bas
 }
 
 func (aul *accountUpdatesLedgerEvaluator) LookupKv(rnd basics.Round, key string) (*string, error) {
-	panic("not implemented")
```

Comment:
Need to implement as migration from sqlite uses `Ledger.accts: accountUpdates` when importing blocks: cf. indexer's [MakeLedger](https://github.com/algorand/indexer/blob/5584c6dca2f2f3a492f7650aeb905ab24ffac682/util/ledger_util.go#L72) opens the ledger of type `Ledger`

Reply:
This gets to my question above about avoiding this cache loading jazz for the indexer.  This code will eventually be needed anyway, so the only bad thing that may come from you doing it with indexer work is that we have to get this code merged into feature branch sooner than your entire indexer work is done.  Nothing wrong with that, I suppose.